test/production/next-config-ts/app/next.config.ts

```diff
@@ -0,0 +1,5 @@
+import { NextConfig } from 'next'
+
+const config: NextConfig = {}
+
+export default config
```

Comment:
IMHO you should add more test cases against different module formats of `next.config.ts`, e.g. export assignment syntax:

```ts
import { NextConfig } = require('next')

const config: NextConfig = {}

export = config
```

Reply:
I will add more test cases. Thanks!